src/ImageSharp/Formats/Tiff/TiffConfigurationModule.cs

```diff
@@ -13,7 +13,9 @@ public void Configure(Configuration configuration)
         {
             configuration.ImageFormatsManager.SetEncoder(TiffFormat.Instance, new TiffEncoder());
             configuration.ImageFormatsManager.SetDecoder(TiffFormat.Instance, new TiffDecoder());
-            configuration.ImageFormatsManager.AddImageFormatDetector(new TiffImageFormatDetector());
+            configuration.ImageFormatsManager.AddImageFormatDetector(new TiffImageFormatDetector(false));
```

Comment:
Is there a way to do this without multiple `TiffImageFormatDetector` instances? This feels like an abuse of the API.

Reply:
really, it's easier